legend-engine-protocol-pure/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/context/AlloySDLC.java

```diff
@@ -20,9 +20,15 @@
 
 public class AlloySDLC extends SDLC
 {
-    @JsonProperty(value = "project", required = true)
+    @JsonProperty(value = "project")
```

Comment:
Could you please deprecate the project variable?

Reply:
Done!